doc/user_manual/existing_interfaces.tex

```diff
@@ -3056,27 +3060,34 @@ \subsubsection{Models}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsubsection{Files}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-The \xmlNode{Files} XML node has to contain all the files required to run the external code  (Cobra-TF).
-For RAVEN coupled with Cobra-TF, only input file(s) (.inp) are needed.
+The \xmlNode{Files} XML node has to contain all the files required to run the external code  (CTF).
+For RAVEN coupled with CTF, there are three input files. CTF input file (.inp) is required by the code. This input file includes all the geometry, boundary and calculation definitions.
 
-Example:
+\noindent There are two additional files (\textit{optional}) that can be used for model parameter perturbation(s) (vuq$\_$param.txt, vuq$\_$mult.txt). These two files can be used to change variables of models embedded in CTF. The ''vuq$\_$param.txt'' file includes parameter values, and ''vuq$\_$mult.txt'' file includes multipliers or additions to parameters. These files are not required by CTF unless a parameter modification is desired. One, both or neither of them can be included in the simulation folder. The code first controls if these files exist and does modifications accordingly if needed.
+
+\noindent
+\textbf{\textit{\nb Currently CTF does not accept any file names other than \textcolor{red}{vuq$\_$params.txt} and \textcolor{red}{vuq$\_$mult.txt} for parameter exposure}}
+
+\noindent Example:
 \begin{lstlisting}[style=XML]
 <Files>
-  <Input name="cobraTF_input" type="">case1.inp</Input>
+  <Input name="CTF_input" type="ctf">case1.inp</Input>
+  <Input name="vuq_param_input" type="vuq_param">vuq_param.txt</Input>
+  <Input name="vuq_mult_input" type="vuq_mult">vuq_mult.txt</Input>
```

Comment:
Explanation is added.

Reply:
There is a small explanation above. What else should I write?

Reply:
That one is too brief. Just add more descriptions. Another question is: Do you really need the ```type```? First, the input file can be identified with the input file extension, i.e. ```.inp```. Second, the ```vuq_param.txt``` and ```vuq_mult.txt``` are hardcoded file name. Since you are using the generic interface parse to parse these files, all these files looks the same from the parse point of view, and there is no meaning to specify the ```type``` here. Please let me know what do you think.

Reply:
Actually now that I think about it, alternative of using the `type` is checking the existence of files whose names are hard coded. So if a user named it wrong, the interface would behave as if it doesn't exist. Furthermore, CTF input file can have any name, so it is not hard coded and I feel like having `type`, is the correct way to distinguish. 

Please let me know about your suggestions.

Reply:
Keep type is file with me, but it seems redundant. As I said, the input file extension is used to identify the input files. Another problems with the type, when you want to perturb additional files which are different from current files, you may need to specify the type for consistent.

Reply:
@wangcj05 I updated the manual and did minor modification to interface. Currently any file can be perturbed by generic interface. The `type` arguments provide users with namecheck. As we discussed in the morning I will find a much better solution to this but I think current version is ready to be used.

I will be glad, if you could review the changes, activate the test and give feedback.